src/main/server.ts

```diff
@@ -33,26 +36,20 @@ function getSSLOptions (): any {
   }
 }
 
-function forkListenerProcesses (numberOfCores: number) {
-  for (let i = 0; i < numberOfCores; i++) {
-    cluster.fork()
+function applicationPort (): any {
+  let port: any = process.env.PORT
+  if (port === undefined) {
+    logger.info(`Port value was not set using PORT env variable, using the default of ${defaultPort}`)
+    port = defaultPort
   }
-  cluster.on('exit', (worker, code, signal) => {
-    logger.info(`Worker ${worker.process.pid} exited with ${code ? code : signal}`)
-  })
+  return port
 }
 
-const testingSupportActivated = toBoolean(config.get<boolean>('featureToggles.testingSupport'))
+const entryPoint = () => listen(app, applicationPort())
```

Comment:
What about `startApplicationFn`, then we have `cluster.run(startApplicationFn)` and `startApplicationFn()`.

The other idea I had was to extract `run()` method to interface and provide two implementations, one which just does `startApplicationFn()` and other one which does the cluster run.  This way in the code we would have if cluster is disabled:

```
ApplicationRunner.run(startApplicationFn)
```

and when cluster enabled

```
ClusteredApplicationRunner.run(startApplicationFn)
```